static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -0,0 +1,157 @@
+
+$accessibility-outline: #ffbf47;
+$correct-answer--background: #aedfb8;
+$user-response-was-right: #43b347;
+$user-response-was-wrong: #bb001a;
+
+.quiz {
+    margin: 0 0 $gs-baseline;
+    padding: 0;
```

Comment:
I think this is my CSS copied from the old quizzes? But in any case, I wonder if it helps to document why we're having to unset/reset declarations like this. It might be because this element inherits other styles, either from classes or by tag name, but you wouldn't know that just looking at this code. Comments aren't ideal, though…